### PR TITLE
docs(charts): align writing

### PR DESCRIPTION
**Pull request by bettiifr**
Edit the doku for the ECharts

### CODE REVIEW

File: packages/documentation/docs/controls/charts/overview.md [hunk 1]
```diff
@@ -6,33 +6,66 @@ import Playground from '@site/src/components/PlaygroundV3';
...
 </div>
 
+![Chart usage guide](https://www.figma.com/design/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?node-id=3532-4181&t=MD9MvUCkoIcmSi8H-4)
+
+## Attributes
+
+| Name           | Description                                                                                                                                                                                   |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Axes**       | Most charts have a horizontal (x-axis) and a vertical (y-axis) axis. These axes are labeled to indicate what data they represent.                                                             |
+| **Scale**      | Each axis has a scale, often marked with periodic graduations and numerical or categorical indications.                                                                                       |
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/charts/overview.md:22]: Is there a simpler way to describe this?
Plus, I think it should be "every axis"

File: packages/documentation/docs/controls/charts/overview.md [hunk 2]
```diff
@@ -6,33 +6,66 @@ import Playground from '@site/src/components/PlaygroundV3';
...
+| **Labels**     | Axes labels describe the dimensions represented, often including units of measurement, e.g. “Distance traveled (m)”.                                                                          |
+| **Grid lines** | Grid lines help in visually aligning data points within the chart.                                                                                                                            |
+| **Legend**     | Legends explain the symbols, colors, or patterns used in the chart to represent different data sets. By clicking on the date in the legend, you can toggle the visibility of the data series. |
+| **Tooltip**    | Provides detailed information about data while hover over the area of interest.                                                                                                               |
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/charts/overview.md:26]: 1. hover -> hovering? (plus, there was a guideline about how we say, and I believe it was just hovering, not hovering hover - but please doublecheck)
2. detailed information -> details?
3. consistency -> Before you started with the entity name, e.g. Tooltips provide ...?!

File: packages/documentation/docs/controls/charts/overview.md [hunk 3]
```diff
@@ -43,9 +76,28 @@ noMargin
...
 <Playground
 height="40rem"
 name="echarts-empty-state"
 noMargin
 examplesByName>
 </Playground>
+
+### Loading indicator
+
+A loading indicator provides users with visual feedback that the chart is being processed and will be displayed shortly. This helps manage user expectations and enhances the overall user experience by preventing confusion or frustration during data loading times.
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/charts/overview.md:90]: Could you please check that again? Generally, you got a point. Maybe you could focus a bit more on the relevant, or unique parts, and instead kick out the chart-unspecific parts? (e.g. "manage user expectations" might not bring a lot of value to the guide)

File: packages/documentation/docs/controls/charts/overview.md [hunk 4]
```diff
@@ -43,9 +76,28 @@ noMargin
...
+
+### Loading indicator
+
+A loading indicator provides users with visual feedback that the chart is being processed and will be displayed shortly. This helps manage user expectations and enhances the overall user experience by preventing confusion or frustration during data loading times. 
+
+### Failure and error message
+
+A failure occurs when no data can be displayed within the chart. This can happen for various reasons, such as connection failure, missing data or other. Error messages have following three elements to move users forward:
+
+- State problem: What happened?​ Add a clear reason for the error, i.e. “No data available”
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/charts/overview.md:96]:
```suggestion
- State problem: What happened?​ Add a clear reason for the error, e.g. “No data available”
```

File: packages/documentation/docs/controls/charts/overview.md [hunk 5]
```diff
@@ -43,9 +76,28 @@ noMargin
...
+- Explain cause: Why did the error appear?​ A clear and concise message explaining why the error happened. "Connection failure"
+- Give solution: What can the user do to proceed?​  
+    Add clear instructions for the user regarding what to do next to resolve the error, e.g. “Try again”
+
+### Missing data points
+
+Indicate missing data with a special visual marker (like a different color or shape) to highlight the gaps without connecting them.
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/charts/overview.md:103]: I would like to **see** something here, let it be a coded or figma visualization

File: packages/documentation/docs/controls/charts/special-chart.md [hunk 6]
```diff
@@ -6,18 +6,11 @@ import Playground from '@site/src/components/PlaygroundV3';
...
-
-## Examples
-
+ECharts offers a wide variety of different chart types and features. Following page deals with some more special chart types and chart features, which might become handy in some use cases. 
 ### Interactive toolbox
 
-Apache ECharts offers a versatile toolbox that enables users to interact with and manipulate charts effectively.
-By default, the toolbox appears in the top right corner of the chart.
-You can customize this toolbox using the `toolbox` option within the option object.
+Apache ECharts offers a versatile toolbox that enables users to interact with and manipulate charts effectively. By default, the toolbox appears in the top right corner of the chart. The toolbox includes various interactive tools (like download, zoom, change type), each designed to enhance the user experience. You can customize this toolbox using the `toolbox` option within the option object.
```
Comment (kathrinschalber) [inline packages/documentation/docs/controls/charts/special-chart.md:12]: maybe use bullets for the tools?

File: packages/documentation/docs/controls/charts/overview.md (hunk 5 above)
Comment (bettiifr) [reply packages/documentation/docs/controls/charts/overview.md:103]: A code example will be added later

**Pull request merged**